api/broker/broker.go

```diff
@@ -0,0 +1,70 @@
+/*
+ *    Copyright 2018 The Service Manager Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package broker
+
+import (
+	"net/http"
+
+	"github.com/Peripli/service-manager/rest"
+)
+
+const (
+	apiVersion = "v1"
+	root       = "service_brokers"
+	url        = "/" + apiVersion + "/" + root
+)
+
+// Routes returns slice of routes which handle broker operations
+func (brokerCtrl *Controller) Routes() []rest.Route {
+	return []rest.Route{
+		{
+			Endpoint: rest.Endpoint{
+				Method: http.MethodPost,
+				Path:   url,
+			},
+			Handler: rest.APIHandler(brokerCtrl.addBroker),
```

Comment:
POST to /v1/service_brokers/ returns all service brokers. If I remove the slash '/' then it's ok.

Reply:
See: http://www.gorillatoolkit.org/pkg/mux#Router.StrictSlash

Reply:
I know that. My thought was that this should be marked as an issue and be fixed soonish